replication-api-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -106,19 +94,19 @@
 
     <service ref="configManager" interface="org.codice.ditto.replication.api.persistence.ReplicatorConfigManager"/>
 
-    <bean id="historyService" class="org.codice.ditto.replication.api.impl.ReplicatorHistoryImpl" init-method="init">
+    <bean id="historyService" class="org.codice.ditto.replication.api.impl.persistence.ReplicatorHistoryManagerImpl" init-method="init">
```

Comment:
These are now the wrong arguments for this class. After this pr there shouldn't be any references to the catalog.